modules/influxdb/influxdb.go

```diff
@@ -47,8 +47,24 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 		}
 	}
 
-	hasInitDb := false
+	if genericContainerReq.WaitingFor == nil {
+		genericContainerReq.WaitingFor = defaultWaitStrategy(genericContainerReq)
+	}
```

Comment:
I agree, I think we should append user-provided wait strategy with what we consider the default one.

Reply:
I don’t agree with this approach for two reasons:

1. It doesn’t allow the caller to specify a fully custom waitStrategy. If, for instance, there’s a change in InfluxDB, the user wouldn’t be able to completely customize their waiting logic.
2. If the caller passes a MultiStrategy, appending could cause issues, as MultiStrategy doesn’t support nesting within another MultiStrategy—from my experience, this setup simply won’t work

Reply:
Interesting topics. About 2, we can maybe open an issue, although supporting multi-level wait strategies could be a very complex scenario not worth it to handle... or maybe it is. I'm not sure and would like to know your opinion.

About 1. You're right: once we couple the module to a given wait strategy, we are coupling the code to a given version of the technology (not only for influx, but for any module), so enabling users to change it sounds sensible to me. Thanks for bringing this thought.

Reply:
Good challenge, I would suggest this is the uncommon case but it would be nice to handle.

So here's an idea which I think handles all of this:
1. Set the default in the creation of `req` to the health check.
2. Have `WithInitDb` override the default with the needed changes

With this the user can still use a `CustomizeRequestOption` latter in the chain to override or write their own `WithInitDb` if that's more appropriate.

Thoughts?

Reply:
I agree. Setting the default to the health check in the `req` creation and then having `WithInitDb` override it as needed will simplify the code. It also keeps the flexibility for users to still use a `CustomizeRequestOption` later in the chain or even write their own WithInitDb. This approach should cover all my concerns.